src/assessments/images/test-steps/image-function.tsx

```diff
@@ -30,12 +30,14 @@ const howToTest: JSX.Element = (
         <br />
         <Markup.Emphasis>
             Notes: (1) If an image has no code to identify it as meaningful or decorative, it will
-            fail an automated check. (2) Assistive technologies will ignore any image coded as
-            decorative, even if it has an accessible name.
+            not be shown here; instead it will fail an automated check. (2) Assistive technologies
```

Comment:
given that we didnt do the exclusion (don't show if it fails image-alt rule) I think we can leave 1 as it was.

Reply:
Updated!